includes/multi-currency/class-settings.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/**
+ * WooCommerce Payments Multi-currency Settings
+ *
+ * @package WooCommerce\Admin
+ */
+
+namespace WCPay\Multi_Currency;
+
+use WCPay\Multi_Currency\Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+if ( class_exists( 'Settings', false ) ) {
```

Comment:
I haven't tested it so it's worth trying it first, but I _think_ we could get the fully qualified class name by using the `::class` keyword here, which would help prevent any possible conflict issues just in case there are any classes called `Settings` defined by any other plugins in use in the site:

```php
if ( class_exists( Settings::class, false ) ) {
```

Reply:
Addressed in: https://github.com/Automattic/woocommerce-payments/pull/2007